elastic/kibana
# Use docLinks plugin to generate server side URL

Resolves: https://github.com/elastic/kibana/issues/109937

In this PR, I'm fixing the `Task Manager detected a degradation in performance` log message to provide the proper versioned link to the task manager health monitoring docs. Prior to this PR, it would always point to main / master docs.

## To verify

1. Turn on debug logging in your kibana.yml file
```
logging:
  loggers:
    - name: plugins.taskManager
      level: debug
```
2. Move this code line outside of the `if (logLevel..` statement => https://github.com/elastic/kibana/blob/4c7ce9d24962913da2207390bc1678280391f9d9/x-pack/plugins/task_manager/server/lib/log_health_metrics.ts#L99
3. Startup Kibana
4. Notice the `Task Manager detected a degradation in performance...` logged
5. Test the URL provided by the log message
